### PR TITLE
add reference to "the principle of least surprise"

### PR DESCRIPTION
**Pull request by sndrs**
this came up in a conversation with @DavidLawes this morning, and seem like good additions to this recommendation

also:

- fixes some whitespace bits (worth viewing diff [without whitespace changes](https://github.com/guardian/recommendations/compare/master...add-the-principle-of-least-surprise?quick_pull=1&w=1))
- rewrite "your" as "our"

### CODE REVIEW

File: coding-with-empathy.md [hunk 1]
```diff
@@ -10,8 +10,10 @@
...
+    -   this will optimise our projects for Guardian engineers
 -   prefer idiomatic patterns and techniques
-    -   this will optimise your code for people who are already familiar with the language it is written in
-- prefer simplicity over sophistication
-    -   this will optimise your code for the next reader
+    -   this will optimise our code for people who are already familiar with the language it is written in
+-   prefer simplicity over sophistication
+    -   this will optimise our code for the next reader
+-   defer to [the principle of least surprise](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)
+    -   our could should make the reader feel capable, not out of their depth
```
Comment (akash1810) [inline coding-with-empathy.md:19]: typo: Should this be "our _code_..."?

**Review by akash1810 — COMMENTED** [review]
Small typo, else LGTM.

File: coding-with-empathy.md [hunk 2]
```diff
@@ -10,8 +10,10 @@
...
+    -   this will optimise our projects for Guardian engineers
 -   prefer idiomatic patterns and techniques
-    -   this will optimise your code for people who are already familiar with the language it is written in
-- prefer simplicity over sophistication
-    -   this will optimise your code for the next reader
+    -   this will optimise our code for people who are already familiar with the language it is written in
+-   prefer simplicity over sophistication
+    -   this will optimise our code for the next reader
+-   defer to [the principle of least surprise](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)
+    -   our code should make the reader feel capable rather out of their depth
```
Comment (SiAdcock) [inline coding-with-empathy.md:19]:
```suggestion
    -   our code should make the reader feel capable rather than out of their depth
```

**Review by SiAdcock — APPROVED** [review]
Small grammatical suggestion, but otherwise awesome 👍

**Pull request merged**